src/System.Threading.Timer/tests/TimerChangeTests.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using Xunit;
+
+public class TimerChangeTests
+{
+    private static void EmptyTimerTarget(Object o) { }
+
+    [Fact]
+    public static void Timer_Change_NegativeTimeSpan_DueTime_Throws()
```

Comment:
Yeah, like I said, I could be wrong :)  Not sure where I got that idea from, but I just tried it and it doesn't appear to matter.  Still, I prefer xunit unit tests as instance rather than static methods.